cocos/core/assets/texture-cube.jsb.ts

```diff
@@ -200,7 +204,7 @@ textureCubeProto._deserialize = function (serializedData: ITextureCubeSerializeD
     const data = serializedData;
     jsb.TextureBase.prototype._deserialize.call(this, data.base, handle);
     this.isRGBE = data.rgbe;
-    this._mipmapMode = parseInt(data.mipmapMode);
+    this._mipmapMode = data.mipmapMode;
```

Comment:
data.mipmapMode is number, no need to parseInt.
Web implementation doesn't have parseInt logic too.